frontends/CMakeLists.txt

```diff
@@ -71,6 +71,10 @@ set (P4_FRONTEND_SRCS
   p4/typeChecking/bindVariables.cpp
   p4/typeChecking/syntacticEquivalence.cpp
   p4/typeChecking/typeChecker.cpp
+  p4/typeChecking/typeCheckDecl.cpp  
+  p4/typeChecking/typeCheckExpr.cpp
+  p4/typeChecking/typeCheckStmt.cpp
+  p4/typeChecking/typeCheckTypes.cpp
```

Comment:
Note that the new header should be added below (in P4_FRONTEND_HDRS) as well.  Not doing so results in it not being linted properly, though it won't cause any build failures.

Reply:
Oh, thanks for catching this, completely forgot about this!